tests/test_configuration.py

```diff
@@ -52,35 +54,35 @@ def test_argument_parsing(self, mock_sys):
             configuration.scan_arguments, ["-DSomeJVMArg", "-DAnotherJVMArg", "-dNotAJVMArg", "-SomeNonsense"]
         )
 
-        mock_sys.argv = ["path/to/scanner/py-sonar-scanner", f"-Dtoml.path={CURRENT_DIR}/resources/pyproject.toml"]
+        mock_sys.argv = [SAMPLE_SCANNER_PATH, f"-Dtoml.path={CURRENT_DIR}/resources/pyproject.toml"]
         configuration.setup()
         self.assertListEqual(
             configuration.scan_arguments,
-            [f"-Dtoml.path={CURRENT_DIR}/resources/pyproject.toml", "-Dsonar.a=b", "-Dsonar.c=d"],
+            ["-Dsonar.a=b", "-Dsonar.c=d", f"-Dtoml.path={CURRENT_DIR}/resources/pyproject.toml"],
```

Comment:
Related to my comment about refactoring, I don't really see why the toml path needs to be passed down to the scanner. It seems only relevant to the wrapper, and therefore I think some of the test cases might need to be changed to reflect this logic.